### PR TITLE
UCT/RC/IFACE: fixed macro

### PR DESCRIPTION
**Pull request by hoopoepg**

### CODE REVIEW

**Comment by hoopoepg** [comment]
@brminich look please

File: src/uct/ib/rc/base/rc_iface.h [hunk 1]
```diff
@@ -388,7 +388,7 @@ typedef struct uct_rc_am_short_hdr {
            void *hdr; \
```
Comment (brminich) [inline src/uct/ib/rc/base/rc_iface.h]: Maybe also  align line 386?

Comment (hoopoepg) [reply src/uct/ib/rc/base/rc_iface.h]: aligned

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6401/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6402/ for details (Mellanox internal link).

**Comment by hoopoepg** [comment]
bot:bgate:retest

**Comment by hoopoepg** [comment]
@yosefe ok to go?

**Pull request merged**